packages/amplify-e2e-tests/package.json

```diff
@@ -25,7 +25,7 @@
   },
   "dependencies": {
     "@aws-amplify/graphql-transformer-core": "1.1.0-cdkv2.1",
-    "amplify-app": "^4.2.39",
+    "amplify-app": "4.3.4-cdkv2.0",
```

Comment:
This will need caret version back when we release GA. https://app.asana.com/0/1202389680978480/1203201977550367/f